server/server/common/queries.py

```diff
@@ -462,6 +461,37 @@
 '''
 
 GLOSSARY = '''
-FOR gloss IN glossary
-    RETURN KEEP(gloss, ['glossword', 'description'])
+LET glossaryitem = (
```

Comment:
please use `snake_case` when naming python and arangodb variables. `glossary_item`